packages/camera/camera_avfoundation/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_avfoundation
 description: iOS implementation of the camera plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/camera/camera_avfoundation
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.9.8+6
+version: 0.10.0
```

Comment:
0.9.9, since this is not a breaking change.

Reply:
Done